binding/nodejs/README.md

```diff
@@ -28,59 +13,51 @@ yarn add @picovoice/pvrecorder-node
 
 ## Usage
 
-Getting the list of input devices does not require an instance:
+To start recording initialize the instance and run `start`:
 
 ```javascript
 const { PvRecorder } = require("@picovoice/pvrecorder-node");
 
-const devices = PvRecorder.getAudioDevices();
+const recorder = new PvRecorder(/*sets to default device*/-1, /*frame length*/ 512);
+recorder.start()
 ```
 
-To start recording initialize the instance and run `start`:
+(or)
+
+Use `get_available_devices()` to get a list of available devices and then initialize the instance based on the index of a device:
 
 ```javascript
-const recorder = new PvRecorder(/*sets to default device*/-1, /*frame length*/ 512);
+
+const { PvRecorder } = require("@picovoice/pvrecorder-node");
+
+const devices = PvRecorder.getAvailableDevices() // select index of device
```

Comment:
'select index of device' here makes me think that `getAvailableDevices()` is going to select the index.